server/szurubooru/func/user_tokens.py

```diff
@@ -0,0 +1,142 @@
+import pytz
```

Comment:
Doesn't this need to go to `requirements.txt` as well?

Reply:
Yes probably.

What's odd is that, I've recently rebuilt my virtualenv with 3.5 since I encountered some odd 3.6 v 3.5 issues related to datetime handling and I didn't explicitly install a pytz version. Perhaps it is coming in as a transient dependency from another library. Still, it should be an explicit library requirement.

Reply:
I've added it to the requirements.txt